bears/haskell/HaskellLintBear.py

```diff
@@ -19,7 +21,9 @@ class HaskellLintBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='hlint')}
+    REQUIREMENTS = {CabalRequirement(
+        package='hlint', version='1.9.35-1'),
```

Comment:
Also @jayvdb do we need DistributionRequirement? We are installing hlint using cabal means shouldn't DistributionRequirement be deprecated?